Sources/KlaviyoSwift/APIRequestErrorHandling.swift

```diff
@@ -112,7 +112,7 @@ func handleRequestError(
         case let .retryWithBackoff(requestCount, totalCount, _):
             requestRetryCount = requestCount + 1
             totalRetryCount = totalCount + 1
-            nextBackoff = getDelaySeconds(for: totalRetryCount)
+            nextBackoff = retryAfter ?? getDelaySeconds(for: totalRetryCount)
```

Comment:
Should we apply jitter to the `retryAfter` too?

Reply:
I suppose if we needed jitter, the server could apply it.

Reply:
@jason-myers-klaviyo do you have any opinion on whether the SDKs should apply a jitter on top of the `Retry-After` value? Does the `Retry-After` just represent the time till the steady state rate limit "resets" or is there more advanced logic to it?

Reply:
Jitter longer than the seconds given would be fine and up to you.  Nice to have I think

Jitter shorter will cause another 429 I think

Reply:
It is just the time until the steady windows reset IIRC

Reply:
Oh ya definitely only _add_ jitter